.gitignore

```diff
@@ -76,3 +76,7 @@ venv.bak/
 
 # Editor Files
 .idea
+
+examples/santa*
+examples/artificial*
+examples/final_individual
```

Comment:
I'm guessing these are some examples that you added in your fork/clone, but don't want to contribute to the main repo (at least not yet 😃 ). If that is correct, then this `.gitignore` change probably makes sense for your fork, but not the main repository.  If not, then can you explain this change a bit more.

Reply:
@erp12 You're right: I was working on an example, which was not yet finished. 

What do you suggest to do now? Should I perform another commit that deletes those lines in the `.gitignore`?

Reply:
Yup, that would be great! Thanks.

Reply:
@erp12 Ok. I've done that. Check it.